tests/estimators/online/test_ukf.py

```diff
@@ -0,0 +1,229 @@
+import numpy as np
+
+from moirae.estimators.online.distributions import MultivariateGaussian, PointEstimate
+from moirae.estimators.online.kalman.unscented import UnscentedKalmanFilter as UKF
+from moirae.models.base import CellModel, InputQuantities, GeneralContainer, HealthVariable, OutputQuantities
+
+
+# Define Lorenz dynamics
+
+class LorenzState(GeneralContainer):
+    x: float
+    y: float
+    z: float
+
+
+class LorenzControl(InputQuantities):
+    sigma: float
+    rho: float
+    beta: float
+    n: float
+
+
+class LorenzOutputs(OutputQuantities):
+    m1: float
```

Comment:
I'm confused, didn't I use to have two values being output?

Reply:
One is the terminal_voltage 🤫